pandas/core/generic.py

```diff
@@ -7872,6 +7872,14 @@ def isna(self: NDFrameT) -> NDFrameT:
         Everything else gets mapped to False values. Characters such as empty
         strings ``''`` or :attr:`numpy.inf` are not considered NA values
         (unless you set ``pandas.options.mode.use_inf_as_na = True``).
+        Note: For :class:`FloatingArray` and float-type
+        :class:`ArrowExtensionArray` the behavior is different, as they
+        use a mask (represented as pd.NA values) to designate missing
```

Comment:
I don't think it is needed here (in this docstring) to explain the underlying implementation (which is strictly speaking not necessarily the cause, for example R also has NaN and NA, but don't use a mask)

Reply:
Sure, makes sense to omit implementation details